components/List/List.js

```diff
@@ -0,0 +1,35 @@
+import {
+  ListContainer,
+  StyledDeleteButton,
+  StyledP,
+  StyledConditionalP,
+  StyledCheckBox,
+  ListWrapper,
+} from "./List.styled";
+
+function List({ toDos, onDeleteToDo }) {
+  return (
+    <>
+      <ListWrapper>
+        <ul>
+          {toDos.map((toDo, index) => (
+            <div key={toDo.id}>
+              <ListContainer>
+                <StyledCheckBox type="checkbox"></StyledCheckBox>
```

Comment:
I think this is not necessary:
```suggestion
                <StyledCheckBox type="checkbox">
```